### ISSUE TITLE
Document installing pyright and provision the nodeenv in one go

### ISSUE THREAD

**Issue by hholst80** [RobertCraigie/pyright-python#178]
How can I install pyright and have the install provision the nodeenv simultaneously?

**Comment by RobertCraigie** [comment]
Hey @hholst80, there isn't a "built-in" way to do this but you can run `pip install pyright && pyright --help` to ensure the nodeenv is cached.

**Comment by hholst80** [comment]
That's exactly what I am doing in the docker build right now. If that's the recommended way, all good.

**Comment by RobertCraigie** [comment]
Yeah, sorry this isn't documented! I'll re-open this to track docs.

**Comment by RobertCraigie** [comment]
Actually now that pyright is bundled by default and you can use the `nodejs` extra to provision node.js as well, I don't think any other docs changes are required :)
